editor/resources/templates/template.compute_program

```diff
@@ -0,0 +1 @@
+program: ""
```

Comment:
It's a bit annoying to have a resource type called `.compute_program`, that points to a `program`, that is a `.cp`, which is short for "compute program". 🤔

If this field were to correspond with how `.vp` is referenced from the Material, it should be `compute_program`. I think the `.compute_program` resource type is named wrong.

Reply:
Yes I think the field name change makes sense, but what should the encompassing resource be called in this case then?
* .compute
* .compute_kernel
* .compute_shader